javascript-data-model/javascript-data-model/order-history.js

```diff
@@ -0,0 +1,36 @@
+const orderHistory = [
+  {
+    orderPlaced: '8/4/2020',
+    total: 34.0,
+    deliveryDate: '8/8/2020',
+    itemName: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr.Axel',
+  },
+
+  {
+    orderPlaced: '7/19/2020',
+    total: 44.53,
+    deliveryDate: '7/20/2020',
+    itemName: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+  },
+
+  {
+    orderPlaced: '7/4/2020',
+    total: 17.22,
+    deliveryDate: '7/7/2020',
+    itemName:
+      'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+  },
+
+  {
+    orderPlaced: '7/3/2020',
+    total: 138.93,
+    deliveryDate: '7/5/2020',
+    itemName: [
+      'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+      'The Art of Sql',
+    ],
+    author: 'Faroult, Stephanie',
```

Comment:
This is a great start! I like that you separated the `itemNames` as indexes in the array... but taking it a step further could help extract out more of the details from the screenshot.

Instead of just the title, you could have an `items` array that houses each `item` as an object. Something like:
```
items: [
{
   title: '',
   author: '',
   price: 10.00,
   returnWindow: '8/4/2020'
},
... // additional item objects
]
```

We'll go over the LFZ solution in Q&A